src/llaminator.ts

```diff
@@ -101,6 +102,13 @@ export class Llaminator {
     const file = (event as CustomEvent).detail as File;
     const blob = new Blob([await file.arrayBuffer()], { type: file.type });
 
+    for (let i = 0; i < easterEggs.length; i++) {
```

Comment:
nit: for..in loop can be more idiomatic: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Reply:
I usually try to avoid for..of in js, because inevitably I will end up using for..in by accident and weird things will start to happen 😞